packages/wallet/ui/src/components/Requests.jsx

```diff
@@ -59,12 +60,24 @@ const RequestsInternal = ({
       return <DappConnection dapp={request.data} key={request.data.id} />;
     }
   };
+
+  const signOffer = () => {
+    const {
+      signers: { interactiveSigner },
+    } = keplrConnection;
+    interactiveSigner
+      .submitSpendAction('{"give": 1, "want": 2}')
+      .catch(err => console.error('TODO: handle sign/broadcast errors', err));
+  };
   return (
     <div className="Requests">
       {requests.length ? (
         requests.map(Item)
       ) : (
         <div className="Empty">
+          <button type="button" onClick={signOffer}>
```

Comment:
I think it's okay to not include this button in the merge, we have the code in other branches to reference

Reply:
ok; it's gone